### ISSUE TITLE
formatTZ returns wrong locale

### ISSUE THREAD

**Issue by plueschpruem** [knowledgecode/date-and-time#53]
```
import date from "date-and-time";
import timezone from "date-and-time/plugin/timezone";
date.plugin(timezone);
import de from "date-and-time/locale/de";
date.locale(de);

const now = new Date();
this.CalendarOptions.weekday = date.format(now, "dddd"); // returns 'Donnerstag'
this.CalendarOptions.weekdayTZ = date.formatTZ(now, "dddd", "Europe/Berlin"); // returns 'Thursday'
```

When using timezones, I cannot get the german locals weekdays from date.formatTZ.
Is that a bug, or am I missing something?

thanks!

**Comment by knowledgecode** [comment]
Thank you for the notice. It is a bug. However, it is not an easy task and will take some time to fix.

**Comment by plueschpruem** [comment]
Oh wow. Once in a while it's not me, but a bug! :-)
I will use a workaround for the time being.

**Comment by knowledgecode** [comment]
This issue has been fixed and published as v2.0.1.
